bin/server/cmd/root.go

```diff
@@ -173,7 +192,30 @@ func start() {
 		return
 	}
 
-	activeProfile := activeProfile(flags.dataDir)
+	var profile server.Profile
+	// This enables backup to cloud, and all backup data will be stored in the supported cloud storage.
+	if flags.backupBucket != "" {
```

Comment:
nit: I may swap the if block order

if flags.backupBucket == "" {
} else {
}

rationales:

1. Avoid negation
2. the block is short, so it's easier to see more context.